lms/templates/instructor/instructor_dashboard_2/membership.html

```diff
@@ -37,6 +37,12 @@
       ${_("You will not get notification for emails that bounce, so please double-check spelling.")} </label>
     <textarea rows="6" name="student-ids" placeholder="${_("Email Addresses/Usernames")}" spellcheck="false"></textarea>
   </p>
+  <p>
+    <label for="reason-field-id">
+      ${_("Enter the reason why the students are Enroll or unEnroll manually.")}
+      ${_("The reason field cannot be left blank.")} </label>
+    <textarea rows="2" id="reason-field-id" name="reason-field" placeholder="${_('Reason')}" spellcheck="false"></textarea>
+  </p>
```

Comment:
Sorry if this was not clear in the story requirement. I think we just need to make the reason be mandatory if the course is a WhiteLabel course (e.g. course_mode price != 0.0 and mode='honor'). I'm not sure we want to make this change to be global on every course.